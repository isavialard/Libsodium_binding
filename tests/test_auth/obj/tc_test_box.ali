V "GNAT Lib v2019"
M P W=b
A -gnatA
A -mtune=generic
A -march=x86-64
P ZX

RN
RV NO_EXCEPTIONS
RV NO_IO
RV NO_STANDARD_STORAGE_POOLS
RV NO_IMPLEMENTATION_PRAGMAS
RV SPARK_05

U test_box%b		tc_test_box.adb		0c5bb383 NE OO SU
W ada%s			ada.ads			ada.ali
Z ada.exceptions%s	a-except.adb		a-except.ali
W ada.text_io%s		a-textio.adb		a-textio.ali
W libsodium_binding%s	libsodium_binding.ads	libsodium_binding.ali
W libsodium_interface%s	libsodium_interface.adb	libsodium_interface.ali

D ada.ads		20190518041255 76789da1 ada%s
D a-except.ads		20190518041255 291912d5 ada.exceptions%s
D a-ioexce.ads		20190518041254 e4a01f64 ada.io_exceptions%s
D a-stream.ads		20190518041255 119b8fb3 ada.streams%s
D a-tags.ads		20190518041256 491b781d ada.tags%s
D a-textio.ads		20190518041256 aef4d56d ada.text_io%s
D a-unccon.ads		20190518041255 0e9b276f ada.unchecked_conversion%s
D interfac.ads		20190518041254 5ab55268 interfaces%s
D i-c.ads		20190518041254 5421119b interfaces.c%s
D i-cstrea.ads		20190518041254 e53d8b8e interfaces.c_streams%s
D libsodium_binding.ads	20190731120158 72a4033a libsodium_binding%s
D libsodium_interface.ads  20190808103816 b75f7039 libsodium_interface%s
D system.ads		20190518041255 4635ec04 system%s
D s-crtl.ads		20190518041254 0ebbdb71 system.crtl%s
D s-exctab.ads		20190518041255 54135002 system.exception_table%s
D s-ficobl.ads		20190518041256 078245e4 system.file_control_block%s
D s-parame.ads		20190518041256 48ec542b system.parameters%s
D s-secsta.ads		20190518041256 20bbe636 system.secondary_stack%s
D s-soflin.ads		20190518041255 a7318a92 system.soft_links%s
D s-stache.ads		20190518041256 a37c21ec system.stack_checking%s
D s-stalib.ads		20190518041256 09bd3940 system.standard_library%s
D s-stoele.ads		20190518041256 2dc34a04 system.storage_elements%s
D s-traent.ads		20190518041254 005bf670 system.traceback_entries%s
D s-unstyp.ads		20190518041255 34867c83 system.unsigned_types%s
D s-wchcon.ads		20190518041255 1b7d22d2 system.wch_con%s
D tc_test_box.adb	20190812101009 675415ed test_box%b
G a e
G c Z b b [test_box standard 7 11 none]
G r c none [test_box standard 7 11 none] [randombytes libsodium_interface 175 14 none]
G r c none [test_box standard 7 11 none] [crypto_box_keypair libsodium_interface 384 14 none]
G r c none [test_box standard 7 11 none] [randombytes libsodium_interface 179 14 none]
G r c none [test_box standard 7 11 none] [crypto_box_easy libsodium_interface 387 14 none]
G r c none [test_box standard 7 11 none] [crypto_box_open_easy libsodium_interface 398 14 none]
X 1 ada.ads
16K9*Ada 20e8 26|3r6 3r24
X 6 a-textio.ads
57K13*Text_IO 727e16 26|3w10 3r28
508U14*Put_Line 26|29s4
X 11 libsodium_binding.ads
7K9*Libsodium_Binding 617e22 26|4w6 4r29
16M9*uint8
19A9*Block8(16M9)<14I9>
X 12 libsodium_interface.ads
5K9*Libsodium_Interface 619e24 26|5w6 5r31
8X4*Crypto_Error 26|25r44
101A9*Box_Secret_Key<11|19A9>(11|16M9)<11|14I9> 26|11r15 13r15
102A9*Box_Public_Key<11|19A9>(11|16M9)<11|14I9> 26|10r15 12r15
105A9*Box_Nonce<11|19A9> 26|15r15
117A9*Plain_Text<11|19A9>(11|16M9)<11|14I9> 26|8r15 14r15
118A9*Cipher_Text<11|19A9>(11|16M9)<11|14I9> 26|9r15
175U14*Randombytes 26|18s4
179U14*Randombytes 26|21s4
384U14*Crypto_Box_Keypair 26|19s4 20s4
387U14*Crypto_Box_Easy 26|22s4
398U14*Crypto_Box_Open_Easy 26|23s4
X 26 tc_test_box.adb
7U11*Test_Box 7b11 31l5 31t13
8a4 M_Alice{12|117A9} 9r28 9r45 14r27 14r44 18m17 22r24 25r22
9a4 C{12|118A9} 22m21 23r33
10a4 PK_Alice{12|102A9} 19m24 23r39
11a4 SK_Alice{12|101A9} 19m34 22r44
12a4 PK_Bob{12|102A9} 20m24 22r36
13a4 SK_Bob{12|101A9} 20m32 23r49
14a4 M_Bob{12|117A9} 23m26 24r13 24r28 25r10
15a4 N{12|105A9} 21m17 22m33 22r33 23r36
24i8 I<long_integer> 25r16 25r30

