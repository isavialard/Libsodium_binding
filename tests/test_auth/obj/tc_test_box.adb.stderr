tc_test_box.adb:7:11: warning: file name does not match unit name, should be "test_box.adb"
